.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: clojure
+lein: lein2
+jdk:
+  - oraclejdk7
+  - openjdk7
```

Comment:
I realize this is probably copied from another project, but ...

* We should add java 8 if possible
* I don't see why we would need to test against two versions of java 7, especially considering we have Jenkins.

If those things both make sense, we can probably just change `oraclejdk7` to `oraclejdk8` and call it good.